internal/cmd/stack/flags.go

```diff
@@ -52,7 +52,12 @@ var flagRequiredRun = &cli.StringFlag{
 
 var flagRun = &cli.StringFlag{
 	Name:  "run",
-	Usage: "`ID` of the run",
+	Usage: "[Optional] `ID` of the run",
+}
+
+var flagRunLatest = &cli.BoolFlag{
```

Comment:
We could instead do this automatically if no `--run` was provided, but I feel like that's kinda unexpected. I'm torn here, so suggestions appreciated.

Reply:
I think for now, let's keep this as an option to explicitly set 👍